deploy/helm/cht-chart/templates/api-deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: api
+  name: cht-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: api
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        cht.service: api
+    spec:
+      containers:
+        - env:
+            - name: BUILDS_URL
+              value: https://staging.dev.medicmobile.org/_couch/builds_4
+            - name: COUCH_URL
+              valueFrom:
+                secretKeyRef:
+                  name: cht-couchdb-credentials
+                  key: COUCH_URL
+            - name: UPGRADE_SERVICE_URL
+              value: http://upgrade-service.{{ .Values.namespace }}.svc.cluster.local:5008
+          image: public.ecr.aws/medic/cht-api:{{ .Values.cht_image_tag }}
```

Comment:
The upgrade service will receive the whole image tag when upgrading, like: public.ecr.aws/medic/cht-api:4.2.0. Can we use that and not hardcode the whole URL.

Reply:
Addressed.